scripts/pylintrc

```diff
@@ -63,3 +63,7 @@ expected-line-ending-format=LF
 
 # Maximum number of characters on a single line.
 max-line-length=130
+
+[MISCELLANEOUS]
+
+notes=XXX
```

Comment:
Please add `TMP`.

Also, if you put it in a separate PR, we could merge it today.

Reply:
Here: https://github.com/ethereum/solidity/pull/14441